visualization/app/codeCharta/util/codeMapHelper.ts

```diff
@@ -25,6 +24,10 @@ export class CodeMapHelper {
 			.map(node => node.data.path)
 	}
 
+	public static setBlacklistFlagByType(node: CodeMapNode, type: BlacklistType, flag: boolean) {
+		type === BlacklistType.exclude ? (node.isExcluded = flag) : (node.isFlattened = flag)
+	}
+
```

Comment:
Isn't a normal if statement nicer to read?

Reply:
When it comes to assignments, probably yes. Will change